features/promotion/managing_promotions/adding_promotion_with_action_in_different_channels.feature

```diff
@@ -1,24 +1,21 @@
 @managing_promotions
-Feature: Adding a new promotion with action configured in different currencies
+Feature: Adding a new promotion with action configured in different channels
     In order to give possibility to pay specifically less price for some goods
     As an Administrator
-    I want to add a new promotion with action configured in different currencies to the registry
+    I want to add a new promotion with action configured in different channels to the registry
 
     Background:
-        Given the store has currency "USD"
-        And the store has currency "GBP"
-        And the store operates on a channel named "Web-US" in "USD" currency
-        And that channel allows to shop using "USD" and "GBP" currencies
+        Given the store operates on a single channel in "United States"
+        And the store operates on a channel named "Web-GB" in "GBP" currency
```

Comment:
`And the store also operates on another channel named ...` - we can reuse this step